.github/workflows/freebsd.yml

```diff
@@ -0,0 +1,37 @@
+name: FreeBSD tests
+
+on:
+    pull_request:
+        types: [labeled, synchronize]
+
+jobs:
+  freebsd:
+    runs-on: macos-latest
+    name: FreeBSD
+    if: contains(github.event.pull_request.labels.*.name, 'tests:full')
+    steps:
+    - uses: actions/checkout@v2
+    - name: Functional tests under FreeBSD
+      uses: vmactions/freebsd-vm@v0.0.8
+      with:
+        usesh: true
+        run: |
+          set -ex
+          freebsd-version
+          mount -o acls /
+          pkg install -y bash unzip rsync ca_root_nss jq fping screen flock
+          mkdir -p /opt/bastion
+          rsync -a . /opt/bastion/
+          fetch https://github.com/ovh/ovh-ttyrec/archive/master.zip
```

Comment:
Master and not last release ?

Reply:
Yes, for now, because master does compile correctly under FreeBSD, and I'd like to avoid releasing a new version just for this ;)